net/dnscrypt-proxy/files/dnscrypt-resolvers.csv

```diff
@@ -1,7 +1,10 @@
 Name,Full name,Description,Location,Coordinates,URL,Version,DNSSEC validation,No logs,Namecoin,Resolver address,Provider name,Provider public key,Provider public key TXT record

```

Comment:
Resolvers list [updates frequently](https://github.com/jedisct1/dnscrypt-proxy/commits/master/dnscrypt-resolvers.csv) and we can update it separately even if dnscrypt version not changed.